types/office-js-preview/index.d.ts

```diff
@@ -18416,7 +18471,8 @@ declare namespace Office {
         set(name: string, value: any): void;
     }
     /**
-     * Provides methods to get and set the appointment sensitivity of a meeting in an Outlook add-in.
+     * Provides methods to get and set an appointment's {@link https://support.microsoft.com/office/4a76d05b-6c29-4a0d-9096-71784a6b12c1 | sensitivity level}
+     * in an Outlook add-in.
```

Comment:
I'm not sure you need this detail about an "Outlook add-in". Isn't that implied by being in Office.js?